invoke-ai/InvokeAI
# chore: remove old web server code and python deps

## What type of PR is this? (check all applicable)

- [x] Tidy

## Description

chore: remove old web server code and python deps
- flask and eventlet are no longer needed as direct dependencies
- remove the old web server entirely (woohoo!)

## Related Tickets & Documents



Supersedes  #4225


## QA Instructions, Screenshots, Recordings



- Manually uninstall the packages removed: `pip uninstall flask flask_cors flask_socketio flaskwebgui eventlet`
- Reinstall the app dependencies: `pip install -e .`
- The app should still run and generate
